SharpPcap/LibPcap/PcapDevice.cs

```diff
@@ -225,6 +225,9 @@ public virtual PacketDotNet.LinkLayers LinkType
         /// </summary>
         public virtual void Close()
         {
+            //Remove event handlers
```

Comment:
why the move of this event handler being cleared out?

Reply:
Clearing of event handler won't take place if device has not been opened, or failed to open.